community.license/pom.xml

```diff
@@ -11,7 +11,7 @@
     <groupId>org.kaazing</groupId>
     <artifactId>community.license</artifactId>
 
-    <name>Kaazing Corporation License</name>
+    <name>Community License</name>
     <url>https://github.com/kaazing/community</url>
```

Comment:
I'm would think we'd still want the Kaazing name attached to things.  I could see "Kaazing Corporation Community License" (to match [enterprise](https://github.com/kaazing/enterprise/blob/develop/enterprise.license/pom.xml)'s reference to "Kaazing Corporation Enterprise License".  Then again, I don't know if we're officially saying things are "community", or if we just want to keep it "Kaazing Corporation License", to sound more official.